src/languages/es.ts

```diff
@@ -3351,6 +3351,10 @@ export default {
             viewUnpaidInvoices: 'Ver facturas emitidas pendientes',
             sendInvoice: 'Enviar factura',
             sendFrom: 'Enviar desde',
+            invoicingDetails: 'Detalles de facturación',
+            invoicingDetailsDescription: 'Esta información aparecerá en sus facturas.',
+            companyName: 'Nombre de la empresa',
+            companyWebsite: 'Sitio web de la empresa',
```

Comment:
I've asked about translations confirmation in the slack: https://expensify.slack.com/archives/C01GTK53T8Q/p1723018638687499